index.html

```diff
@@ -262,6 +265,38 @@ <h4 class="modal-title" id="singUpModal">Zajerestruj się</h4>
     </div>
   </div>
 
+  <div class="modal fade" id="addRestaurantModal" tabindex="-1" role="dialog" aria-labelledby="addRestaurantLabel">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="myModalLabel">Dodaj Restauracje</h4>
+        </div>
+        <div class="modal-body">
+          <form id="addRestaurantModalForm">
```

Comment:
form needs `action` and `method` attributes to know where to send data on submit